### ISSUE TITLE
Module crashes Freescout once enabled

### ISSUE THREAD

**Issue by cwoelkers** [vladaman/freescout-saml2#1]
After enabling the module I refreshed the page and received the "Something has gone wrong" error.
Checking the logs found the following:
[2022-01-21 13:56:49] production.ERROR: main(): Failed opening required '/var/www/html/Modules/Saml/Providers/../vendor/autoload.php' (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 64): main(): Failed opening required '/var/www/html/Modules/Saml/Providers/../vendor/autoload.php' (include_path='.:/usr/share/php') at /var/www/html/Modules/Saml/Providers/SamlServiceProvider.php:5)
[stacktrace]
#0 {main}
"} 

Using Freescout 1.7.27, the latest version as of this post.

**Comment by freescout-helpdesk** [comment]
Do you actually have `'/var/www/html/Modules/Saml/vendor/autoload.php' ` file on your server?

**Comment by cwoelkers** [comment]
No. That file should come from this module's codebase as it resides within the Saml module directory.
